gwcs/tags/geometry_models.py

```diff
@@ -13,7 +13,7 @@
 class DirectionCosinesType(GWCSTransformType):
     name = "direction_cosines"
     types = [ToDirectionCosines, FromDirectionCosines]
-    version = "1.0.0"
+    version = "1.1.0"
```

Comment:
This looks like it might break backwards compatibility with ASDF files written with `direction_cosines-1.0.0` tags.  Since the `direction_cosines-1.0.0` schema still exists, would it be good to make sure the tag can use it?

@eslavich, does adding a `supported_versions = ["1.0.0", "1.1.0"]` class attribute solve this?

(Same comment for the rest of the Types modified here.)

Reply:
The asdf library will use this class for 1.0.0 if no other option is available, and we've been relying on that behavior in both astropy and asdf itself, so I think this is probably okay as-is for now.  The new AsdfConverter API will not provide the same fallback behavior so when that comes along we'll need to explicitly list each supported version.